doc/api/test.md

```diff
@@ -327,8 +330,8 @@ test('Test 4', async (t) => {
 
 Test name patterns can also be specified using regular expression literals. This
 allows regular expression flags to be used. In the previous example, starting
-Node.js with `--test-name-pattern="/test [4-5]/i"` would match `Test 4` and
-`Test 5` because the pattern is case-insensitive.
+Node.js with `--test-[name/skip]-pattern="/test [4-5]/i"` would match `Test 4`
```

Comment:
I don't know if we should make this particular change because new users are likely to copy and paste that.  I'd leave this as is, or if you really want, use only the skip flag here.